server/src/session.ts

```diff
@@ -600,9 +632,8 @@ export class Session {
     if (this.ivy && isConfiguredProject(project)) {
       // Keep language service disabled until ngcc is completed.
       project.disableLanguageService();
-      this.connection.sendNotification(notification.RunNgcc, {
-        configFilePath: project.getConfigFilePath(),
-      });
+      // Do not wait on this promise otherwise we'll be blocking other requests
+      this.runNgcc(project.getConfigFilePath());
```

Comment:
Do you think it would be good to add an [unhandled promise rejection handler](https://nodejs.org/api/process.html#process_event_unhandledrejection)?